### ISSUE TITLE
Navbar toggler needs to be bigger

### ISSUE THREAD

**Issue by jonnebonde** [NoroffFEU/agency.noroff.dev#1032]
The navbar toggler is only clickable on the span bars, it should be clickable on the whole button for better usage and accessibility.
